packages/p2p/src/components/buy-sell/buy-sell-header.jsx

```diff
@@ -57,6 +58,14 @@ const BuySellHeader = ({ is_visible, table_type, setTableType }) => {
             buy_sell_store.setItems([]);
             buy_sell_store.setIsLoading(true);
             buy_sell_store.loadMoreItems({ startIndex: 0 });
+
+            const interval = setInterval(() => {
+                buy_sell_store.getWebsiteStatus();
```

Comment:
Why do we need this at an interval?

Reply:
We need to manually update the local currencies every minute as the API is not subscribable.